File: Week_5/practice_1_my_abs_function.py
# Practice_1 Question: Write a function named my_abs() that receives an integer value as parameter and returns the absolute value of that. Absolute value means if the value is negative it will be converted to positive value. Then print the value in the main function.

"""
sample input        output
5                   5
-7                  7
"""


# Function to calculate absolute value
def my_abs(n):
    if n < 0:
        absolute = n * (-1)
    else:
        absolute = n
    return absolute


n = int(input())        # Input value

# Calculate and print the absolute value
absolute_value = my_abs(n)

print(absolute_value)

File: Week_5/practice_1_my_abs_function.cpp
// Practice_1 Question: Write a function named my_abs() that receives an integer value as parameter and returns the absolute value of that. Absolute value means if the value is negative it will be converted to positive value. Then print the value in the main function.

/* 
sample input        output
5                   5
-7                  7
 */


#include <bits/stdc++.h>
using namespace std;

int my_abs(int n)
{
    int absolute;

    if (n < 0)
    {
        int new_absolute = n * (-1);
        absolute = new_absolute;
    }
    else
    {
        absolute = n;
    }

    return absolute;
}

int main()
{
    int n;
    cin >> n;

    int absolute_value = my_abs(n);
    cout << absolute_value << endl;  

    return 0;
}
